### ISSUE TITLE
Eclipse doesn't start

### ISSUE THREAD

**Issue by Everworkx** [adoptium/adoptium-support#590]
### Please provide a brief summary of the bug

-

### Expected Results

-

### Actual Results

If i open eclipse, i start and shutdown before i can see Dashboard

### What Java Version are you using?

java version "1.8.0_341" Java(TM) SE Runtime Environment (build 1.8.0_341-b10) Java HotSpot(TM) 64-Bit Server VM (build 25.341-b10, mixed mode)

### What is your operating system and platform?

Windows 10 64bit

### Did it work before?

```shell
it worked for 3 days and now the error comes every times
```

### Did you test with other Java versions?

```shell
no
```

### Relevant log output

```shell

```
[hs_err_pid6932.log](https://github.com/adoptium/adoptium-support/files/9678959/hs_err_pid6932.log)

**Comment by jerboaa** [comment]
> java version "1.8.0_341" Java(TM) SE Runtime Environment (build 1.8.0_341-b10) Java HotSpot(TM) 64-Bit Server VM (build 25.341-b10, mixed mode)

This isn't a temurin build --^, fwiw.

Going by the hs_err file the actual version in use is JDK 17:

```
OpenJDK Runtime Environment Temurin-17.0.4.1+1 (17.0.4.1+1) (build 17.0.4.1+1)
```

Crash is in c2:

```
# Problematic frame:
# J 7638 c2 jdk.internal.jimage.ImageLocation.decompress(Ljava/nio/ByteBuffer;I)[J (116 bytes) @ 0x0000028e33d498ff [0x0000028e33d49620+0x00000000000002df]
```

**Comment by Everworkx** [comment]
i have installed jdk: 

java version "17.0.4.1" 2022-08-18 LTS
Java(TM) SE Runtime Environment (build 17.0.4.1+1-LTS-2)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.4.1+1-LTS-2, mixed mode, sharing)

no changes. eclipse crash after startup

**Comment by LucasBQ** [comment]
It's an amd graphic driver error I had the same problem in intellij
 OpenJDK Runtime Environment Temurin-17.0.4.1+1.

 solved by installing Adrenalin 22.5.1 Recommended (WHQL) (downgrade)
